docker-compose.yml

```diff
@@ -15,5 +15,6 @@ services:
     environment:
       LAALAA_API_HOST: "https://laa-legal-adviser-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
       DJANGO_SETTINGS_MODULE: "fala.settings.local"
+      FALA_NO_MAP: "fala.settings.local"
```

Comment:
This doesn't look right to me - I believe it will set the environment variable to `fala.settings.local`, which is not what we're after. (Although things will still work because Django will take in preference the value from local.py, so it will still work if the user created that file). So I suggest we omit this line, and just let the user edit the value in local.py.

Reply:
Yeah, after doing some testing on a different branch, I can confirm it's not needed.